cmd/aries-js-worker/index.html

```diff
@@ -11,11 +11,11 @@
 <html>
```

Comment:
should have license header comments.

Reply:
I'm confused - it does have the license header comments lines 1->9.